src/Api/Request/CancelService.php

```diff
@@ -28,7 +28,7 @@
 
 class CancelService
 {
-    const CANCEL_API = '/api/Payment/v1/Transaction/Cancel';
```

Comment:
you sure its not needed?

Reply:
const TEST_API = 'https://test.saferpay.com/api/';
    const API = 'https://www.saferpay.com/api/';
    
    the urls already have /api/ , I believe it would be https://test.saferpay.com/api/api/Payment/../ with previous uri